packages/editor/src/components/post-visibility/index.js

```diff
@@ -17,6 +17,15 @@ import { __experimentalInspectorPopoverHeader as InspectorPopoverHeader } from '
 import { visibilityOptions } from './utils';
 import { store as editorStore } from '../../store';
 
+/**
+ * PostVisibility component.
```

Comment:
We need to remove this `PostVisibility component.` from every component because the generated docs include the component's name, so it's kind of duplicate..